spec/spec_helper.rb

```diff
@@ -6,6 +6,11 @@
   Coveralls.wear!
 end
 
+if ENV['SCRUTINIZER']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.